src/OrchardCore.Modules/OrchardCore.DataProtection.Azure/OrchardCore.DataProtection.Azure.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.DataProtection.AzureStorage" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.Environment.Shell.Abstractions\OrchardCore.Environment.Shell.Abstractions.csproj" />
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.Modules.Abstractions\OrchardCore.Modules.Abstractions.csproj" />
+    <ProjectReference Include="..\OrchardCore.DataProtection\OrchardCore.DataProtection.csproj" />
```

Comment:
You shouldn't have to reference this module. Modules should not have to reference other modules, even if they depend on it.

Reply:
Yeah I wasn't too sure whether or not I needed to do this-

On one hand, I'm not using any classes or interfaces from this module, so it'll compile without the reference.

On the other hand, this feature is dependent on a feature defined in the referenced project. I thought that if I were to install this module as a nuget package, I'd like the dependent features to come with it too.

Reply:
It's a logical dependency. You don't want to ship your module with the other one. You just expect the module to already be there and be enabled.

Reply:
OK, I thought that having a missing dependency would be an issue (like O1), but I realize now that's not the case.

I'll update and push up.